src/xcode/ENA/ENA/Source/Scenes/TraceLocations/TypeSelection/Cells/SelectTraceLocationTypeCell.swift

```diff
@@ -0,0 +1,73 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+
+class SelectTraceLocationTypeCell: UITableViewCell, ReuseIdentifierProviding {
+
+	// MARK: - Init
+
+	override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+		super.init(style: style, reuseIdentifier: reuseIdentifier)
+		setupView()
+	}
+
+	@available(*, unavailable)
+	required init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+
+	// MARK: - Internal
+
+	func configure(cellModel: TraceLocationType) {
+		titleLabel.text = cellModel.title
+		subTitleLabel.text = cellModel.subtitle
+	}
+
+	// MARK: - Private#
+
+	private let titleLabel = ENALabel()
+	private let subTitleLabel = ENALabel()
+
+	private func setupView() {
```

Comment:
In the dark mode video it looks like we don't have the correct background color on these cells.

Reply:
good point, I change the color to cellBackground
![Simulator Screen Shot - iPhone 12 Pro - 2021-03-22 at 16 29 34](https://user-images.githubusercontent.com/2675578/112015189-e2cef280-8b2b-11eb-9ab4-7791628feab4.png)